internal/pkg/template/manifest/file.go

```diff
@@ -81,7 +81,7 @@ func saveFile(fs filesystem.Fs, content *file) error {
 
 func (f *file) validate() error {
 	ctx := context.WithValue(context.Background(), validator.DisableRequiredInProjectKey, true)
-	if err := validator.ValidateCtx(ctx, f, "dive", ""); err != nil {
+	if err := validator.New().ValidateCtx(ctx, f, "dive", ""); err != nil {
```

Comment:
Na nekritickych miestach, ako je napr. nacitanie/ulozenie manifestu, som to nekomplikoval.